Makefile

```diff
@@ -77,13 +79,13 @@ open:
 
 ## Start the hugo server for live editing
 run:
```

Comment:
let's make `docker/build` a dependency of `run`

Reply:
```
run: docker/build
```

Reply:
You'll need to define these:

https://github.com/cloudposse/terraform-root-modules/blob/master/Makefile#L1-L5